index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+    <header id="header">
+      <div class="logo">
+        <img src="logo.png" alt="" id="header-img">
+      </div>
+    
+    <nav id="nav-bar">
+      <ul>
+        <li><a id="nav-link" href="#"> Features</a>
+        <li><a  id="nav-link" href="#"> How it works</a>
+        <li><a  id="nav-link" href="#"> Pricing</a>
+      </ul>
+      
+    </nav>
+
+    </header>
+    
+    
```

Comment:
there is unneeded space between the HTML tags.

Reply:
- [ ] [Required] We can also say why this change is important I suggest.

Reply:
there is unneeded space between the HTML tags. This will help to keep your code clean and more readable for anyone who would like to contribute also for professional purposes.